core/jvm/src/main/scala/zio/sql/expr.scala

```diff
@@ -246,45 +259,60 @@ trait ExprModule extends NewtypesModule with FeaturesModule with OpsModule {
     }
   }
 
-  object FunctionDef {
+  sealed case class FunctionDefWithDelimiter[-A, +B](name: FunctionName, delimiter: String) extends FunctionDef[A, B] {
```

Comment:
What is the purpose of `delimiter`? Can you help me understand this a bit more?

Reply:
Standard function in SQL looks like this: `SELECT func(param1, param2)` e.g. `SELECT date_part('minutes', source)` while `extract` is different, it takes two parameters, but it looks like this: `SELECT extract(minutes from source)`. So for "standard" function delimiter is `,` while for extract function it is ` FROM `.

Reply:
This looks like a very special case. Maybe we could solve it at the `renderRead` level?

Reply:
You mean something like idea in #309 (extension to `Expr` with custom renderer for it) or just simple `if` checking the name of the function in the `renderRead` ;) ?